clarifai/models/model_serving/models/default_test.py

```diff
@@ -0,0 +1,227 @@
+import dataclasses
+import inspect
+import logging
+import os
+import unittest
+
+import numpy as np
+
+from ..model_config.triton_config import TritonModelConfig
+from .output import (ClassifierOutput, EmbeddingOutput, ImageOutput, MasksOutput, TextOutput,
+                     VisualDetectorOutput)
+
+PREDEFINED_TEXTS = ["Photo of a cat", "A cat is playing around"]
+
+PREDEFINED_IMAGES = [
+    np.zeros((100, 100, 3), dtype='uint8'),  #black
+    np.ones((100, 100, 3), dtype='uint8') * 255,  #white
+    np.random.uniform(0, 255, (100, 100, 3)).astype('uint8')  #noise
+]
+
+
+class DefaultTestInferenceModel(unittest.TestCase):
+  """
+  This file contains 2 test cases of:
+  ...
+  """
+  __test__ = False
+
+  def triton_get_predictions(self, input_data):
+    """Call InferenceModel.get_predictions method
+
+    Args:
+        input_data (Union[np.ndarray, str]):
+          if model receives image or vector then type is `np.ndarray`. Otherwise `string`
+
+    Returns:
+       One of types in models.output
+    """
+    return inspect.unwrap(self.triton_model.inference_obj.get_predictions)(
+        self.triton_model.inference_obj, input_data)
+
+  def _get_preprocess(self):
+    """ preprocess if input is image """
+    if "image" in self.triton_model_input_name:
+      h, w, _ = self.triton_model_config.input[0].dims
+      if h > -1 and w > -1:
+        import cv2
```

Comment:
any chance we can use Pillow instead of cv2? cv2 is a headache to install sometimes; I would try to avoid it if possible.

Reply:
I see `opencv-python` already listed in requirements.txt of the package, it would be installed together with `clarifai` and input is always `np.ndarray` so that I prefer opencv